.github/process_commit.py

```diff
@@ -72,13 +59,14 @@ def get_pr_merger_and_labels(pr_number: int) -> Tuple[str, Set[str]]:
     return merger, labels
 
 
-def get_pr_reviewers(pr_number: int) -> Set[str]:
-    # See https://docs.github.com/en/rest/reference/pulls#list-reviews-for-a-pull-request
-    data = query_torchvision(f"pulls/{pr_number}/reviews", accept="application/vnd.github.v3+json")
-    return {review["user"]["login"] for review in data if review["state"] == "APPROVED"}
-
-
 if __name__ == "__main__":
     commit_hash = sys.argv[1]
-    users = find_responsible_users(commit_hash)
-    print(", ".join(sorted([f"@{user}" for user in users])))
+    pr_number = get_pr_number(commit_hash)
+    if not pr_number:
+        exit
```

Comment:
Shouldn't this be 

```suggestion
        sys.exit(0)
```

?